### ISSUE TITLE
Num labels in librispeech_train_10_1008.subwords is not 1008

### ISSUE THREAD

**Issue by magdacisowska** [TensorSpeech/TensorFlowASR#210]
Number of labels in https://github.com/TensorSpeech/TensorFlowASR/blob/main/vocabularies/librispeech/librispeech_train_10_1008.subwords is not 1008, but 751.
This way I'm not even able to test the trained rnn_transducer model. Can we get the missing vocabulary?

**Comment by usimarit** [comment]
@magdacisowska Did you try loading the file with `SubwordFeaturizer` and see the property `num_classes`?

**Comment by jaehwlee** [comment]
> @magdacisowska Did you try loading the file with `SubwordFeaturizer` and see the property `num_classes`?

It works to me! Change the line number 84 in 'examples/rnn_transducer/test.py' like this.

![image](https://user-images.githubusercontent.com/33409264/134441933-3c890bb9-0df5-4dcc-a044-a53b2cf6aa67.png)

**Comment by usimarit** [comment]
I'll close the issue here since I haven't seen the problem. Feel free to reopen if you still see the issue, make sure you show me the last layer's shape (aka logits layer that produces probs).
